cyclus_input/reactors/Arkansas_Nuclear_1.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation>
```

Comment:
I've removed the simulation tag since reactor.xml files only have one child under root. Thank you :)